front/app/containers/InitiativesShowPage/index.tsx

```diff
@@ -30,9 +33,28 @@ const InitiativesShowPage = () => {
   const initiativesEnabled = useFeatureFlag({ name: 'initiatives' });
   const { slug } = useParams() as { slug: string };
 
-  const { data: initiative } = useInitiativeBySlug(slug);
+  const { data: initiative, status, error } = useInitiativeBySlug(slug);
+
+  if (!initiativesEnabled) {
+    // Ideally, this is covered by status === 'error' but currently there
+    // a bug (in the BE?) that still shows this page to people with URL, even
```

Comment:
On the BE, we don't usually check if a feature is allowed/enabled. We usually just check if a user is allowed to see some resources (e.g., initiatives) according to their role. So, I suppose it's fine to handle it on the FE as you did.